h2o-persist-gcs/build.gradle

```diff
@@ -0,0 +1,21 @@
+description = "H2O Persist GCS"
+
+//apply plugin: "io.spring.dependency-management"
+
+dependencies {
+    compileOnly project(":h2o-core")
```

Comment:
I think this is incorrect, "compileOnly" is analogous to "provided" in maven. It is a set of libraries that are inherently present on the classpath when the application is run in the target environment. For example, you mark the application server libraries as "compileOnly".

I would not use "compileOnly" in this context.

Reply:
My thoughts were exactly as yours. And I thought this is the context where compileOnly is OK to put. In my head, H2O-Core is always on the classpath and makes no sense for GCS to be ran otherwise.

Thanks for reviewing it. I marked this change on purpose in separate commit.